tests/deployment/test_client_server.py

```diff
@@ -85,6 +82,10 @@ def test_client_server_sklearn(
 ):
     """Tests the encrypt decrypt api."""
 
+    if get_model_name(model_class) == "KNeighborsClassifier":
+        # https://github.com/zama-ai/concrete-ml-internal/issues/4014
```

Comment:
please add `FIXME:` before any issue links + a  small comment explaining what needs to be done / fixed

Reply:
alright